demisto_sdk/commands/zip_packs/packs_zipper.py

```diff
@@ -0,0 +1,202 @@
+import logging
+import os
+import sys
+from contextlib import contextmanager
+from pathlib import Path
+
+import click
+from demisto_sdk.commands.common.constants import PACKS_DIR
+from demisto_sdk.commands.common.content.objects.pack_objects.pack import Pack
+from demisto_sdk.commands.common.tools import arg_to_list
+from demisto_sdk.commands.create_artifacts.content_artifacts_creator import (
+    IGNORED_PACKS, ArtifactsManager, ContentObject, create_dirs, delete_dirs,
+    dump_pack, zip_pack_zips, zip_packs)
+
+EX_SUCCESS = 0
+EX_FAIL = 1
+
+
+class PacksZipper:
+
+    def __init__(self, pack_paths: str, output: str, content_version: str, zip_all: bool, quiet_mode: bool = False):
+        self.artifacts_manager = PacksManager(
+            pack_paths=pack_paths,
+            artifacts_path=output,
+            content_version=content_version,
+            all_in_one_zip=zip_all,
+            quiet_mode=quiet_mode,
+        )
+
+    def zip_packs(self):
```

Comment:
Why ?
it's not the same method
and even so - it's play like a manager that call to other methods and do something else in addition

Reply:
It is not instantly clear which `zip_packs` is being called in the `PacksDirsHandler` (which is in the same file) IMO.
I think that for future debugging purposes it might be wise to distinct this method from the imported one.

But it is not major - I leave it to @teizenman to decide the course of action here.

Reply:
Hi @roysagi 
@yaakovi say it's not real issue and can remain with this name
so please approve if you don't have any other comment
Thanks!!